src/components/Register/detailsCard.js

```diff
@@ -1,53 +1,79 @@
-import React from 'react'
-import Config from 'config'
-import Button from 'components/Button'
+import React from 'react';
+import Config from 'config';
+import Button from 'components/Button';
 
 export default class DetailsCard extends React.Component {
-	constructor(props) {
-		super(props)
-		this.handleChange = this.handleChange.bind(this);
-	}
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+    this.state = {
+      passwordLength: 1,
+    };
+  }
 
-	handleChange(e) {
-		if (this.props.onChange)
-			this.props.onChange(e.target.name, e.target.value);
-	}
+  handleChange(e) {
+    if (this.props.onChange) {
+      this.props.onChange(e.target.name, e.target.value);
+      const changed = e.target.name;
+      const changedLength = e.target.value.length;
+      if (changed === 'password') {
+        changedLength < 10 ? this.setState({
```

Comment:
Put the ternary inside the object passed to setState to avoid this awkward syntax of calling a function with a different argument and using its side effect as a result

Reply:
I agree with this! It looks a little messy right now